### PR TITLE
Update mg annotation docs

### LINKED ISSUE #5: mg_annotation - Kaitlyn
Follow the template below for the index.rst file that is in the docs folder of the repo for your workflow:
Title of Workflow
(Image)
Workflow overview
Workflow availability
Requirements for Execution
Hardware requirements
Workflow dependencies
Software
Database(s)
Sample dataset(s)
Input
Output
Version history
Point of contact

If you have updated figures please put the editable components of the figures [here](https://drive.google.com/drive/folders/1MEmTMLMPeDMQaF-LYnpJswjVmn9MUFTX) as well as in the documentation 
Please follow the following format for updating your documentation:

### PR DESCRIPTION
**Pull request by kaijli**
genomad not added, updated readme and index.rst

### CODE REVIEW

**Comment by aclum** [comment]
The formatting of sub bullets especially of the workflow dependencies is not correct. see how things are in main [right now](https://github.com/microbiomedata/mg_annotation/blob/master/docs/index.rst). same for the sample datasets section.

**Comment by aclum** [comment]
We could list out the database versions.

File: README.md [hunk 1]
```diff
@@ -1,44 +1,48 @@
 ## Workflow for Metagenome annotation                                                                                                                                                                                                                      
-This workflow is based on the JGI/IMG annotation pipeline ([details](https://github.com/kellyrowland/img-omics-wdl)). This is still in progress.  It takes assembled metagenomes and generates structrual and functional annotations.
+This workflow takes assembled metagenomes and generates structural and functional annotations. It is based on the JGI/IMG annotation pipeline (`details <https://github.com/kellyrowland/img-omics-wdl>`_) and uses a number of open-source tools and databases to generate the structural and functional annotations.
```
Comment (aclum) [inline README.md]: kellys repo is very old. We can replace this with Marcel's repo which is public (https://code.jgi.doe.gov/img/img-pipelines/img-annotation-pipeline/)  or the publication (https://journals.asm.org/doi/10.1128/msystems.00804-20)

File: README.md [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
- - tRNAscan-SE >= 2.0 (GNU GPL v3)
- - Infernal 1.1.2 (BSD)
- - CRT-CLI 1.8 (Public domain software, last official version is 1.2, I made changes to it based on Natalia's and David's suggestions)
- - Prodigal 2.6.3 (GNU GPL v3)
- - GeneMarkS-2 >= 1.07 ([Academic license for GeneMark family software](http://topaz.gatech.edu/GeneMark/license_download.cgi))
- - Last >= 983 (GNU GPL v3)
- - HMMER 3.1b2 (3-clause BSD, I am using [Bill's thread optimized hmmsearch](https://github.com/Larofeticus/hpc_hmmsearch))
- - SignalP 4.1 (Academic)
- - TMHMM 2.0 (Academic)
+   - Conda (3-clause BSD)
```
Comment (aclum) [inline README.md]: I think the extra space here is messing up the sub bullets

File: README.md [hunk 3]
```diff
@@ -1,44 +1,48 @@
...
+   - Prodigal 2.6.3_patched (GNU GPL v3)
+   - GeneMarkS-2 >= 1.25 ([Academic license for GeneMark family software](http://topaz.gatech.edu/GeneMark/license_download.cgi))
+   - Last >= 1456 (GNU GPL v3)
+   - HMMER 3.1b2 (3-clause BSD, [thread optimized hmmsearch](https://github.com/Larofeticus/hpc_hmmsearch))
+ 
 
 #### Databases used (+ their licenses):
- - Rfam (public domain/CC0 1.0; [more info](http://reusabledata.org/rfam)
- - KEGG (paid subscription, getting KOs/ECs indirectly via IMG NR; [more info](http://reusabledata.org/kegg-ftp)
+ - Rfam (public domain/CC0 1.0; [more info](http://reusabledata.org/rfam))
```
Comment (aclum) [inline README.md]: issue with sub bullet format and more info hyperlinks

File: README.md (hunk 2 above)
Comment (aclum) [reply README.md]: disregard, this only applies to the index.rst

File: README.md (hunk 3 above)
Comment (aclum) [reply README.md]: disregard for the readme, this comment applies to the index.rst

**Comment by aclum** [comment]
Formatting looks great now. Can you add IMG-NR to the database dependencies. I'm getting the licensing info from Marcel.

**Comment by kaijli** [comment]
> Formatting looks great now. Can you add IMG-NR to the database dependencies. I'm getting the licensing info from Marcel.

https://github.com/microbiomedata/mg_annotation/blob/b505d10aa98370094105c72f6afed92943b49df8/docs/index.rst?plain=1#L49

https://github.com/microbiomedata/mg_annotation/blob/b505d10aa98370094105c72f6afed92943b49df8/README.md?plain=1#L45

these don't cut it for adding img-nr?

**Review by aclum — APPROVED** [review]

**Pull request merged**